core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -108,12 +104,15 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.setFirstState = async function(stateName) {
-    await initialStateSelectOption(stateName).click();
+    await action.click(
+      'Initial State Select Option', initialStateSelectOption(stateName));
```

Comment:
Can you include the value of `stateName` in the message so we know which state option we're trying to click?

Reply:
Done